api/admin/controller.py

```diff
@@ -1435,7 +1434,10 @@ def reset_adobe_id(self, authenticator=None):
         # Wipe the Patron's 'identifier for Adobe ID purposes'.
         for credential in AuthdataUtility.adobe_relevant_credentials(patron):
             self._db.delete(credential)
-        return Response(unicode(_("Success")), 200)
+        return Response(
+            unicode(_("Adobe ID for patron %(patron_username)s has been reset.", patron_username=patron.username)),
```

Comment:
@adriana-alter @leonardr I talked to Amy about this and it'll be better to pass a more detailed message from the server than "Success" and let the UI display that.